docs/developer/release/prepare.md

```diff
@@ -0,0 +1,42 @@
+# Prepare build environments
+
+## macOS
+
+### Initial setup
+
+- Ensure that the build machine has:
+  - Apple-trusted `Developer ID Application: Freedom of the Press Foundation (94ZZGGGJ3W)` code-signing certificates installed
+- Apple account must have:
+  - A valid application password for `notarytool` in the Keychain. You can verify
+    this by running: `xcrun notarytool history --apple-id "<email>" --keychain-profile "dz-notarytool-release-key"`. If you don't find it, you can add it to the Keychain by running
+    `xcrun notarytool store-credentials dz-notarytool-release-key --apple-id <email> --team-id <team ID>`
+    with the respective `email` and `team ID` (the latter can be obtained [here](https://developer.apple.com/help/account/manage-your-team/locate-your-team-id))
+  - Agreed to any new terms and conditions. You can find those if you visit
+    https://developer.apple.com and login with the proper Apple ID.
+
+### On each release
+
+- [ ] Upgrade "Command Line Tools" from "System Settings -> Software Update", from an account with admin privileges.
+- [ ] Upgrade Xcode from the App Store, from an account with admin privileges.
+- [ ] Update Docker Desktop and Podman Desktop to the latest versions.
+- [ ] Update Python to the latest supported version, following our [instructions](../python.md)
+
+## Windows
+
+The Windows release is performed in a Windows 11 virtual machine (as opposed to a physical one).
+
+### Initial Setup
+
+- Download a VirtualBox VM image for Windows from here: https://developer.microsoft.com/en-us/windows/downloads/virtual-machines/ and import it into VirtualBox. Also install the Oracle VM VirtualBox Extension Pack.
+- Install updates
+- Install git for Windows from https://git-scm.com/download/win, and clone the dangerzone repo
+- Follow the Windows build instructions in `BUILD.md`, except:
+  - Don't install Docker Desktop (it won't work without nested virtualization)
+  - Install the Windows SDK from here: https://developer.microsoft.com/en-us/windows/downloads/windows-sdk/ and add `C:\Program Files (x86)\Microsoft SDKs\ClickOnce\SignTool` to the path (you'll need it for `signtool.exe`)
```

Comment:
This seem to be a repetition of what's currently in the `BUILD.md` file, which has a ["if you want to build the installer" section](https://github.com/freedomofpress/dangerzone/blob/main/BUILD.md#if-you-want-to-build-the-windows-installer) which I believe should go away and be replaced with these instructions instead.

I didn't get why you're mentioning here "except install the Windows SDK from here": is the link different from what is currently in this other section? Probably we could do some more refactoring here to avoid duplicates.

Reply:
_(Just an FYI, those instructions have not changed from the ones in the previous `RELEASE.md`.)_

This requires a bit more thought, perhaps on a separate issue. On `BUILD.md` we ask people to install .NET and Wix. On `prepare.md`, we point people to follow the instructions on `BUILD.md` plus installing Windows SDK. I _think_ there's no overlap here, because we don't copy the instructions, but we point to them. Still, I don't grok these instructions completely, so I can't be totally sure if they are up to date or not.

Reply:
It most likely is an avenue that some our power users follow, given that we had a PR by a user for WiX stuff. So, I would suggest keeping it here still.